plugins/auth-backend/src/identity/types.ts

```diff
@@ -52,7 +53,7 @@ export type TokenIssuer = {
  */
 export type StoredKey = {
   key: AnyJWK;
-  createdAt: moment.Moment;
+  createdAt: DateTime;
```

Comment:
I realize that this was a moment object before, but I'm wondering why the interface isn't just a standard Date object? Sure we can use luxon for parsing etc but seems like it should be abstracted away. If consumers need to do date manipulation they can choose to do it on the date object or wrap with luxon again. Thoughts?

Reply:
@andrewthauer Do you want me to change to a standard Date object or should we wait for more input?

Reply:
Seems appropriate to me, but maybe @backstage/maintainers care to weigh in?

Reply:
Yeah probably makes more sense to stick to a JS Date here, as this is a bit of a public API. There's also some trouble with the DB layer though, seems like sqlite provides a timestamp as a string, while postgres provides a Date object. With postgres the date object ends up being invalid because the date object ends up being stringified and the parsing then fails.

The broader issue there is the error handling though. Luxon doesn't throw on invalid errors, which is a bit unexpected tbh. It's possible to change that behaviour with this:

```ts
import { Settings } from 'luxon'
Settings.throwOnInvalid = true
```

... which is less than stellar 😅 

Tbh I think we probably bite the bullet and validate all parsed dates by checking `.isValid` and throwing with a message built with `. invalidReason` and `.invalidExplanation`.